octofree/main.py

```
@@ -109,23 +117,134 @@ def main():
+                logging.info(f"✅ [STARTUP RECOVERY] Session recovery complete - added {len(missing_from_tracking)} session(s)")
+            
+            # Summary
+            total_stored = len(startup_scheduled_sessions) + len(startup_past_sessions)
+            logging.info(f"✅ [STARTUP UPDATE] Bayesian update complete: {len(startup_current_sessions)} on website, {total_stored} total stored ({len(startup_scheduled_sessions)} scheduled, {len(startup_past_sessions)} past)")
+            logging.info("────────────────────────────────────────────────────────────────────")
+            
+        else:
+            logging.warning("🔴 [STARTUP SCRAPE] Failed to fetch website data for Bayesian update - proceeding with stored data")
+            
+    except Exception as e:
+        logging.exception(f"🔴 [STARTUP SCRAPE] Error during Bayesian update: {e} - proceeding with stored data")
```

Ruff (0.14.1)

Redundant exception object included in `logging.exception` call

(TRY401)